kotlin/samples/hello-compose-binding/build.gradle.kts

```diff
@@ -28,6 +30,9 @@ android {
 }
 
 apply(from = rootProject.file(".buildscript/configure-compose.gradle"))
+tasks.withType<KotlinCompile> {
+  kotlinOptions.apiVersion = "1.3"
+}
```

Comment:
The compiler the plugin is built against is in some weird liminal state between 1.3 and 1.4, so these warnings are safe to ignore, this suppresses them.

Reply:
Seems like this comment should be in the gradle file.

Reply:
Yep

Reply:
Done.